misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordConverter.java

```diff
@@ -179,7 +180,7 @@ public static JsonToRecordResponse convert(String jsonString, String recordName,
                 Map<String, Schema> fields = schema.getValue().getProperties();
```

Comment:
There is a raw usage of the paramterized type error. Shall we fix that?

Reply:
I did get what you mean in this comment. Could please explain a bit more about what I have to do here?